episodes/15-carpentries.md

```diff
@@ -259,13 +259,13 @@ Select one role from the list below that interests you. Using the the descriptio
   would like to see that are not listed? Note that, too!
 
 1. Executive Council
-2. Mentors
-3. Instructor Trainers
-4. Lesson Developers
-5. Code of Conduct Committee
-6. Instructor Development Committee
-7. Community Facilitators
-8. Maintainers
+2. Instructor development
+3. Trainers
+4. Lesson developers
+5. Curriculum advisors
+6. Lesson maintainers
+7. Regional Coordinators
+8. Code of Conduct Committee
```

Comment:
```suggestion
2. Instructor Trainers
3. Lesson developers
4. Curriculum advisors
5. Lesson maintainers
6. Code of Conduct Committee
```

Reply:
I think this list may be more accurate. Instructor development has been retired, and while there are still people who are serving as regional coordinators I do not think this is continuing as a formal role.